src/View/Helper/PaginatorHelper.php

```diff
@@ -412,10 +419,16 @@ public function sort($key, $title = null, array $options = [])
 
         $sortKey = $this->sortKey($options['model']);
         $defaultModel = $this->defaultModel();
+        $model = Hash::get($options, 'model', $defaultModel);
```

Comment:
Call to `Hash::get()` is not necessary, can just do `$model = $options['model'] ?: $defaultModel;`